static/images/loading.svg

```diff
@@ -0,0 +1,4 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24">

```

Comment:
You can add svgs on `site/_includes/icons` and then import them like `{% include icons/loading.svg %}`